src/components/Layout/Layout.jsx

```diff
@@ -25,43 +25,36 @@ const Layout = ({ children, username, avatar }) => (
       <>
         <SiteHeader
           logo={data.site.siteMetadata.logo}
-          logoDestination="http://example.com"
+          logoDestination="https://www.edx.org/"
```

Comment:
I know the ticket didn't specify but this should also be configurable as `data.site.siteMetadata.siteUrl`. It's less critical right now but when we think about supporting Open edX in the future they would need to override this. We would also want to potentially override in in stage/sandboxes/etc

This should be a relatively small change here but you'll also need to add edx.org as a default to `gatsby-config.js` (until gatsby is fully wired up to wagtail)

Reply:
Thanks @georgebabey. I added data.site.siteMetadata.siteUrl to logoDestination and added siteUrl in the graphql query on Layout.jsx. In the gatsby-config.js I included edx.org. Let me know if that's what you're looking for. I'm not sure how gatsby works when it is wired to wagtail, so let me know if it needs to be written differently.

Reply:
That is perfect. We have a separate story to wire up gatsby to wagtail so that will be handled later with that work.